tests/main/snapd-snap/task.yaml

```diff
@@ -155,6 +155,9 @@ execute: |
       if ! os.query is-trusty && ! os.query is-xenial; then
         systemctl status snapd.apparmor.service
       fi
+      if os.query is-ubuntu; then
+        systemctl status apparmor.service
```

Comment:
When I was researching the issue, initially the apparmor.service was active, but when I restarted it it failed, could be  better to restart it and check it is active after that?

Reply:
Also, this test is not being executed in ubuntu-core which is where we reproduced the issue. Perhaps we could have a new test so we also validate the beta images.